### PR TITLE
[AIRFLOW-1261] clarify the docs for the CLI's airflow run --local flag

### PR DESCRIPTION
**Pull request by mistercrunch**
### Description
Clarifying documentation following an exchange on the mailing list title "Subdag failure" on 6/1/17

### Tests
Build

### CODE REVIEW

**Comment by aoen** [comment]
LGTM after moved to right section

File: airflow/bin/cli.py [hunk 1]
```diff
@@ -1200,6 +1200,17 @@ class CLIFactory(object):
         'local': Arg(
             ("-l", "--local"),
             "Run the task using the LocalExecutor", "store_true"),
```
Comment (aoen) [inline airflow/bin/cli.py:1202]: Should be moved to the # run section

File: airflow/bin/cli.py [hunk 2]
```diff
@@ -1200,6 +1200,17 @@ class CLIFactory(object):
...
+        'run_local': Arg(
```
Comment (saguziel) [inline airflow/bin/cli.py:1203]: Doesn't adding this make it a dupe since there's stuff about this above

**Comment by saguziel** [comment]
Maybe mention LocalTaskJob

**Pull request closed without merging**